docs/using/security-non-events.md

```diff
@@ -69,3 +99,7 @@ Some bugs blocked include:
 * [CVE-2015-3214](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-3214), [4036](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-4036): These are bugs in common virtualization drivers which could allow a guest OS user to execute code on the host OS. Exploiting them requires access to virtualization devices in the guest. Sandstorm hides direct access to these devices. Interestingly, these seem to be cases where Sandstorm's sandbox is "more secure" than a VM, going against common wisdom that VMs are "more secure" than containers.
 * [CVE-2016-0728](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-0728): Use-after-free caused by crafted `keyctl()` calls could lead to privilege escalation. Sandstorm disables `keyctl()` inside the sandbox using seccomp.
 
+Bugs *not* mitigated:
+
+* [CVE-2014-9090](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-9090), [9322](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-9322): AKA "BadIRET". According to Andy Lutomirski, who discovered and fixed the bug (and who wrote Sandstorm's seccomp filter): "Very hard to exploit from inside Sandstorm, but it just might have been possible using a bizarre vector. Certainly the standard exploit would not work."
```

Comment:
...though I guess that's for a section called "Your computer" not "Linux kernel".

## Your computer and physical reality

We can't protect against people with:

- A screwdriver and physical access, willing to remove your hard drive

- Rowhammer

- People looking over your shoulder with a video camera and watching you type your password